src/pages/dev/send-tokens/interchain-tokens/create-token.mdx

```diff
@@ -6,6 +8,10 @@ The Interchain Token Service is deployed to [`0xB5FB4BE02232B1bBA4dC8f81dc24C269
 
 You can create a new Interchain Token through the [Interchain Portal](https://testnet.interchain.axelar.dev/), or by building a custom ERC-20 token and deploying it with a Mint/Burn token manager on all chains. These tokens will be accessible on multiple chains, allowing for seamless interaction on each chain by utilizing familiar methods such as send, transfer, and approve -- just as with any standard ERC-20 token.
 
+<Callout emoji="🚨">
+  **NOTE:** The security of your token is limited to the security of the chains it integrates with. Since blockchains can have different security practices, we recommend verifying each source chain and source contract address before integration.
```

Comment:
ITS does the verification of contract and addresses in this context. Perhaps we could say "we recommend doing due diligence on any chains you wish to deploy your token to" or similar?